### PR TITLE
4 get/api/articles

### PR DESCRIPTION
**Pull request by LukeRiches**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -41,15 +41,74 @@ describe('GET /api', () => {
...
 });
 
+describe('GET /api/articles', () => {
+    test('should respond with an articles array of article objects ', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+            const articles = res.body.articles;
+            console.log(res.body, "res.body");
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: remove console.logs before committing

File: __tests__/app.test.js [hunk 2]
```diff
@@ -41,15 +41,74 @@ describe('GET /api', () => {
...
+            })
+        });
+    });
+    test('should be sorted by date in descending order', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+            const articles = res.body.articles;
+            console.log(articles);
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: See above

File: __tests__/app.test.js [hunk 3]
```diff
@@ -41,15 +41,74 @@ describe('GET /api', () => {
...
+            expect(articles).toBeSortedBy('created_at')
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: Have a look at the docs for toBeSortedBy, I think created_at could be sorted by ascending by default doing it like this (....I think😅)

File: db/Controllers/articles-controller.js [hunk 4]
```diff
@@ -13,4 +13,20 @@ function getArticleByID(req, res, next){
...
+function getArticles(req, res, next){
+    // console.log(req.query, "req.querys");
+
+    if(Object.keys(req.query).length >= 1){
+        return res.status(400).send({msg : "No queries have been declared yet"})
+    }
+
+    fetchArticles()
+    .then((articles)=>{
+        res.status(200).send(articles)
```
Comment (Beth-McCormack) [inline db/Controllers/articles-controller.js]: It is best practice to send responses in an object

**Review by Beth-McCormack — COMMENTED** [review]
Nice work! I've left a couple of comments 🙂 Feel free to merge once you've had a look

File: __tests__/app.test.js (hunk 3 above)
Comment (LukeRiches) [reply __tests__/app.test.js]: Just realised I misread what order to put them in 😅, at first I put them In order of creation but I've amended it now thanks for spotting that

**Pull request merged**